bin/update_readme

```diff
@@ -38,7 +38,7 @@ $sectionTemplate = <<<EOF
         %PROVIDER_NAME%:
             # must be "%PROVIDER_NAME%" - it activates that type!
             type: %PROVIDER_NAME%
-            # add and configure client_id and client_secret in parameters.yml
+            # add and configure client_id and client_secret in parameters.yml or .env* files
```

Comment:
Let's go full "new Symfony" with this :) 

```suggestion
            # add and set these environment variables in your .env files
```

Reply:
Hi @weaverryan , I thought about this initially. But I noticed that `oauth2-client-bundle` supports Symfony v3.4. And this version still uses parameters.yml.

Reply:
It does - but you're own your own as far as documentation goes :). It's impossible to write good documentation for multiple versions at once. For example, just mentioning parameters.yml here isn't enough - 3.4 doesn't use the `env()` syntax at all! Better to be clear and on the latest version